app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @HERMON-1995 great work in your models, however, you forgot to reference the user class and also you used user instead of author when referencing the user , your association should look like ` belongs_to :author, class_name: 'User' `

Reply:
Thank you for pointing out the oversight in my models. You are absolutely right, the associations should reference the author instead of user and include the class_name option to specify the associated class. I have implemented it. 😊